src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -496,7 +496,7 @@ def remove_subvolume_snapshot(self, **kwargs):
             with open_volume(self, volname) as fs_handle:
                 with open_group(fs_handle, self.volspec, groupname) as group:
                     with open_subvol(self.mgr, fs_handle, self.volspec, group, subvolname, SubvolumeOpType.SNAP_REMOVE) as subvolume:
-                        subvolume.remove_snapshot(snapname)
+                        subvolume.remove_snapshot(snapname, force)
```

Comment:
Is the "force" snapshot removal option being tested? Do we need a new test for this?

Reply:
Done